src/init/features/project.ts

```diff
@@ -32,6 +32,11 @@ export interface ProjectInfo {
   location?: string; // maps to FirebaseProjectMetadata.resources.locationId
 }
 
+/**
```

Comment:
wait so we're doing `jsdoc` now?  IMO the comment "Transforms project metadata to a ProjectInfo object." is useful but the rest is just a restatement of the type signature.

Reply:
I want to turn down this rule to "doc only exported functions"... let me dig for a second...

Reply:
Oh yeah, I can do this. Winning.